### ISSUE TITLE
Design Drupal Interfaces to support Vet Centers Outstation enhancements

### ISSUE THREAD

**Issue by thejordanwood** [department-of-veterans-affairs/va.gov-cms#18125]
Blocked by #20212

## Description
For the [Vet Center 2.0 initiative](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17661), front end designs are being updated to solve problems that relate to Vet Center branding, events, and satellite locations. After the FE designs are finalized, Drupal changes will need to be made to accommodate these updates. 

## Design
[Vet Center Figma file](https://www.figma.com/file/EVd3q06ukAbVS61Q8MwRAT/Vet-Centers?type=design&node-id=1%3A900&mode=design&t=Wqtm0fJVxdIkoObB-1)

## Drupal Considerations

- [ ] Parity between Node Edit and  Node View?
- [x] Dashboard experience
- [ ] Outstations
- [ ] Main location
- [ ] Locations page

## Previous issues to address
- [ ] #4903
- [ ] #6156
- [x] #10297
- [ ] New default introduction text from figma? #4704
- [ ] #10301

## Suggestions from [previous dashboard research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/cms/research/vet-center)

- [ ] Use a different label for "Link text" (e.g. Hyperlink) and reinforce the explanation with examples.
- [ ] Display information about when national content was last updated.
- [ ] Differentiate main VC from other locations.
- [ ] Add "scroll to top" functionality to the forms.
- [ ] Provide visual cue regarding content that was updated, but not saved.
- [x] Adjust "Visitor information" label and/or be specific about what information is to be edited on this category.
- [ ] Make it clear that changing the operating status does not influence Location and Contact information information.
- [ ] Remove this information from the operating status flow or place Operating Status on its own block.
- [ ] Clarify what is there to edit when it comes to services at a VC.

## Acceptance Criteria

- [x] Proposal(s) for overall flow is documented and shared with Product and engineeer
- [x] Designs are documented in Figma (as needed)
- [x] Review with UX Lead, Product, Engineering, Content, and Accessibility
- [ ] Follow up tickets are created

**Comment by jilladams** [comment]
@thejordanwood (@Agile6MSkinner )I think we handled all these Drupal changes in other tix already and this ticket could theoretically be closed, unless we're trying to maintain Figma source of truth docs around Drupal UI?

**Comment by davidmpickett** [comment]
@jilladams That is true for the [Vet Center differentiation](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17893) Epic, but there will be more significant Drupal changes necessary for the [Satellite Locations](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17693#top) epic

**Comment by davidmpickett** [comment]
Hey @davidmpickett take a look at #4643 to see if any previously documented issues are still relevant and/od fixable as part of this redesign

**Comment by davidmpickett** [comment]
Considerations for the dashboard
- Services button goes to services for main vet center specifically (though seems like the link is no longer jumping down)
- Outstations don't stand out among satellite locations
- A vet center can have multiple outstations
- Outstations will also have services

![Image](https://github.com/user-attachments/assets/648730e3-53ed-4d8d-abe9-70878c352683)

**Comment by davidmpickett** [comment]
Make sure to address #10297

**Comment by davidmpickett** [comment]
Migrated old Dashboard designs to [Figma file](https://www.figma.com/design/qqQhab9SGhvDNZgRIdWS7g/Vet-Centers-forms?node-id=175-1282)

**Comment by davidmpickett** [comment]
Shared in UX sync and got feedback
- Going to do a mock up of tiles for Main vet center and link collection for outstation
- Going to talk about ways to avoid making links very wordy - bypass link block
-

**Comment by jilladams** [comment]
Other UX sync notes of note: 
* Pending Michelle's thoughts, we may want to get input from our lead Vet Center editor who attended the brainstorming session recently, as a barometer for resistance to change / sentiment around utility of the current dashboard.
* Michael flagged: given the other high value things we could deliver, we may want to take a minimalist approach to changing the current design for Main Vet Centers, on level of effort vs. value for what we could deliver with our time grounds. TBD more product conversations.

**Comment by davidmpickett** [comment]
End of Sprint update, I have been swamped with supporting other tickets and was not able to make progress on revising this before my PTO. I will pick it up when I return on 3/25

**Comment by davidmpickett** [comment]
@FranECross @mmiddaugh @Agile6MSkinner Link to the dashboard design exploration in Figma https://www.figma.com/design/qqQhab9SGhvDNZgRIdWS7g/Vet-Centers-forms?node-id=180-382

**Comment by davidmpickett** [comment]
Shared on slack with context from UX sync https://dsva.slack.com/archives/C0FQSS30V/p1743107514498129

**Comment by davidmpickett** [comment]
**Quick win Recommendation**:
- Put Outstations in a new table separate from other satellite locations - [Design 5](https://www.figma.com/design/qqQhab9SGhvDNZgRIdWS7g/Vet-Centers-forms?node-id=186-4306&t=LcKKXICCM1Wqi3En-4)

**Long term recommendation**
- Move towards expandable accordions to allow quick access to features without information overload as in design 6
- [Collapsed](https://www.figma.com/design/qqQhab9SGhvDNZgRIdWS7g/Vet-Centers-forms?node-id=197-3628&t=LcKKXICCM1Wqi3En-4)
- [Expanded](https://www.figma.com/design/qqQhab9SGhvDNZgRIdWS7g/Vet-Centers-forms?node-id=197-1805&t=LcKKXICCM1Wqi3En-4)

**Comment by mmiddaugh** [comment]
I agree with the recommendation of [Design 5](https://www.figma.com/design/qqQhab9SGhvDNZgRIdWS7g/Vet-Centers-forms?node-id=186-4306&t=LcKKXICCM1Wqi3En-4) as a quick win - with one caveat. I'm concerned it will exacerbate the known problem of editors failing to see satellite locations because of their position on the page. Is there a way to quickly improve this without an entire design overhaul? (i.e. jump links?)
